src/platform/amd/renoir/lib/dai.c

```diff
@@ -54,7 +54,7 @@ static struct dai spdai[] = {
 		.drv = &spdai_driver,
 	}
 };
-
+#ifdef ACP_BT_ENABLE
```

Comment:
how is this set?

Reply:
should this be part of an earlier patch?

Reply:
BT Dai is not the default option on Renoir platforms(HW dependency). 
based on config "CONFIG_AMD_BT" defined in renoir_defconfig, we will enable the ACP_BT_ENABLE macro, which in turn enable the BT Dai initializations.